CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Improvements
 - **General**: Add more events for user checking ([#796](https://github.com/kedacore/keda/issues/3764))
+- **General**: Add parameter queryParameters in prometheus-scaler (N/A)
```

Comment:
Please, create an issue to track this feature and link it here 🙏 
```suggestion
- **General**: Add parameter queryParameters in prometheus-scaler (N/A)
```

Reply:
Hi @JorTurFer ,

I opened issue #4962

Reply:
Cool, link it please in the Changelog as well

Reply:
```suggestion
- **General**: Add parameter queryParameters in prometheus-scaler ([#4962](https://github.com/kedacore/keda/issues/4962))
```